cypress/automated-tests/blind/successful-blind-submission.cy.js

```diff
@@ -148,16 +148,17 @@ describe('blind successful renewal submission - without an email address', () =>
     cy.get('#element48').type(applicantZipCode).blur();
   });
 
-  it("skips the demographic section and agrees to the rules", function() {
-    cy
-      .get('[data-field-code="RulesAndConditionsCheckbox"]')
-      .contains('I agree')
-      .click();
-      
-    cy.get('.form-submit-button').click();
-
-    cy
-      .get('#thank-you-text', { timeout: 15000 })
-      .should('contain', 'Application Submitted');
-  });
+  // Disable submission
+  //it("skips the demographic section and agrees to the rules", function() {
```

Comment:
suggestion: generally not the best idea to keep "dead" code around, because if we tried to introduce it again its more likely to not make sense given other changes that have occurred. I'd say this should just be deleted.